keps/sig-node/4742-node-labels-downward/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: KEP Template
+kep-number: 4742
+authors:
+  - "@cmwylie19"
+  - "@docandrew"
+  - "@jamesallen-atx"
+  - "@thefutoneng"
+owning-sig: sig-node
+participating-sigs:
+status: provisional
+creation-date: 2024-07-01
+reviewers:
+  - TBD
+approvers:
+  - TBD
+
+see-also:
+
+replaces:
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.31"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.31"
+  # beta: "v1.32"
+  # stable: "v1.33"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: TBD
+    components:
+      - kubelet
```

Comment:
I don't believe we need a gate for this feature, so long as it only affects `kubelet`.

Reply:
We use feature gates in Kubelet.

Reply:
Yah, gates are there in case someone finds a nasty bug and needs to quickly disable some new functionality.